### ISSUE TITLE
Fix ParasiticEnergy

### ISSUE THREAD

**Issue by mactyr** [energyPATHWAYS/energyPATHWAYS#19]
Doing some recon for #12 revealed that there is a bug that prevents ParasiticEnergy objects from loading from the database (using sqlite the loading fails silently, but the problem causes an error in postgres which is how I noticed it). Fixing the loading error was trivial, but it revealed a number of knock-on bugs related to using ParasiticEnergy objects that need to be fixed. This ticket is to track the rehabilitation of ParasiticEnergy.

**Comment by BenEvolved** [comment]
Michael,

Sorry, this got wrapped into a larger merge but I'd like you to check that parasitic energy calculations are now working. Specifically, if you check  model.demand.sectors[1].subsectors[34].parasitic_energy you should see the appropriate values (sum  them if most of the values show as 0),

**Comment by mactyr** [comment]
Ok, running against 11787713, doing a run to 2016 just for subsector 34, I get:

```
In [7]: model.demand.sectors[1].subsectors[34].parasitic_energy.sum()
Out[7]: 
2000    63189584.681023
2001    63969039.708040
2002    64716735.828040
2003    65448077.091517
2004    66171107.066809
2005    66906805.317299
2006    67668595.406498
2007    68456552.239426
2008    69192482.839005
2009    69843575.147991
2010    70440879.465128
2011    70960683.868712
2012    71440013.977913
2013    71997569.563409
2014    72553726.293303
2015    73140650.051893
2016    73714981.402522
dtype: float64
```

So this looks ok, and was fixed somewhere between 32d2ba7d76b1297ba1bf73e67fd1f9e2f9434a21 and 11787713, inclusive. Since the fix is already merged in, I will close this ticket and delete my fix_parasitic_energy_19 branch, which is no longer needed.
